public/game/shared/csgo/protobuf/cstrike15_usermessages.proto

```diff
@@ -752,8 +978,3 @@ message CCSUsrMsg_ClientInfo
 {
 	optional int32 dummy = 1;
 }
-
-message CCSUsrMsg_ServerRankRevealAll
-{
-	optional int32 seconds_till_shutdown = 1;
-}

```

Comment:
If this is gone, it probably makes sense to remove our SETUP_MESSAGE call for it as well.

Reply:
It's still there, it just received another field, so I moved it above the comment saying that the data seems to be irrelevant.

Reply:
Makes sense. Thanks